content/en/docs/concepts/staged-update.md

```diff
@@ -4,23 +4,71 @@ description: Concept about Staged Update
 weight: 13
 ---
 
-While users rely on the `RollingUpdate` rollout strategy to safely roll out their workloads, 
-there is also a requirement for a staged rollout mechanism at the cluster level to enable more controlled and systematic continuous delivery (CD) across the fleet.
-Introducing a staged update run feature would address this need by enabling gradual deployments, reducing risk, and ensuring greater reliability and consistency in workload updates across clusters.
+While users can rely on the `RollingUpdate` rollout strategy in the placement to safely roll out their workloads,
+many organizations require more sophisticated deployment control mechanisms for their fleet operations.
+Standard rolling updates, while effective for individual rollout, lack the granular control and coordination capabilities needed for complex, multi-cluster environments.
+
+## Why Staged Rollouts?
+
+Staged rollouts address critical deployment challenges that organizations face when managing workloads across distributed environments:
+
+### Strategic Grouping and Ordering
+
+Traditional rollout strategies treat all target clusters uniformly, but real-world deployments often require strategic sequencing. Staged rollouts enable you to:
+
+- **Group clusters by environment** (development → staging → production)
+- **Prioritize by criticality** (test clusters before business-critical systems)
+- **Sequence by dependencies** (deploy foundational services before dependent applications)
```

Comment:
Isn't this handled by placement instead of updateRun?

Reply:
that's defined in the updateRunStrategy, right?